app/src/main/res/values/colors.xml

```diff
@@ -1,14 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="accent">@*android:color/accent_device_default_light</color>
-    <color name="primary">@*android:color/primary_device_default_settings_light</color>
-    <color name="primaryDark">@*android:color/primary_dark_device_default_settings_light</color>
-    <color name="background">@*android:color/background_device_default_light</color>
-    <color name="actionBarPrimary">@*android:color/primary_device_default_light</color>
-    <color name="statusBarColor">@*android:color/primary_device_default_settings_light
-    </color>
-    <color name="red">@*android:color/error_color_device_default_dark</color>
-    <color name="ic_launcher_background">@*android:color/accent_device_default_light</color>
+    <!-- AOSP colors -->
+    <!-- https://android.googlesource.com/platform/frameworks/base/+/refs/tags/android-14.0.0_r1/core/res/res/values/colors_device_defaults.xml#35 -->
+    <color name="accent">@android:color/system_accent1_600</color>
```

Comment:
How does this work when the accent color is changed in theme picker or through wallpaper? Also, what's the difference to that `@*android:` way of specifying colors?

Reply:
It uses a new accent color then. `@*android:` is resolved at runtime if I remember correctly and the said accent colors are system only.

Reference:

- [R.color](https://developer.android.com/reference/android/R.color)
- [Dynamic color](https://m3.material.io/styles/color/dynamic-color/user-generated-color)